docs/resources/s3_bucket_policy.md

```diff
@@ -16,7 +16,7 @@ Manages **S3 Buckets policies** on IonosCloud.
 ```hcl
 
 resource "ionoscloud_s3_bucket_policy" "example" {
-  bucket = "example"
+  bucket = ionoscloud_s3_bucket.example.name
```

Comment:
@cristiGuranIonos if you want to use `bucket = ionoscloud_s3_bucket.example.name`, then please add a s3 bucket resource to this example (when giving examples, we are doing this for all the resources that have depend on other resources). Otherwise, you can leave it as `bucket = "example"`, the name has to be unique globally anyways.

Reply:
could use "your-bucket-name-here" to prompt users to set their own values and not rely on placeholder since it's unique

Reply:
done